api/README.md

```diff
@@ -144,3 +143,36 @@ The list of available handlebars variables:
   - `name`: The service name which is equivalent to the network alias
   - `port`: The exposed port of the service
   - `type`: The type of service. For example, `instance`, `replica`, `app-companion`, or `service-companion`.
+
+## Hooks
+
+Hooks can be used to manipulate the deployment before handing it over to actual infrastructure. For example, based on the app name you can decide to reconfigure your services to use a different DBMS so that you are able to verify that your services work with different DBMSs.
```

Comment:
How about:
```
Hooks can be used to manipulate the deployment before handing it over to actual infrastructure. Hooks are called for each service and can manipulate each services configuration. For example, based on the app name you can decide to reconfigure your services to use a different DBMS so that you are able to verify that your services work with different DBMSs. Or based on the service name you can specify manipulation for a specific service.
```
I do not need a new review for this.

Reply:
> Hooks are called for each service

That's not quite true because a hook is called once with all service configurations as an argument. I'll hope my new version clarifies that a little bit better.